### DISCUSSION TITLE
Odroid C4, how to set up booting from an external drive

### DISCUSSION THREAD

**Discussion by jusik0** [johang/sd-card-images#120] [Q&A]
Hello everyone,
I have an Odroid C4 and it runs from an SD card.
Where and how to set up booting root partition from an external drive - USB?
Regards

**Comment by johang** [comment]
You can't. Booting from USB is not supported by Amlogic boot ROM, so you'd need to boot U-boot from either MMC/eMMC and then force U-boot to load kernel and initrd from USB.

**Reply by jusik0** [reply]
@johang
I know that an external drive cannot be bootable.
 How to force U-boot to load kernel and initrd from USB?
Where are the boot files in this image?
There is no system map, grub, fstab is empty...
Sorry, maybe this is a stupid question...
Regards

**Reply by jusik0** [reply]
I found the answer.
Thank you for your work.
Kind regards
